ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -322,29 +366,27 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-confidential-trustedlaunch
-- as: azure-ipi-des-mini-perm-fips-amd-f28
-  cron: 26 14 2 * *
+- as: azure-ipi-des-mini-perm-arm-f28
+  cron: 3 13 7 * *
   steps:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
       ENABLE_DES_DEFAULT_MACHINE: "true"
-      FIPS_ENABLED: "true"
+      OCP_ARCH: arm64
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-disk-encryption-set-minimal-permission
-- as: azure-ipi-des-mini-perm-amd-mixarch-f28
-  cron: 43 15 30 * *
+- as: azure-ipi-des-mini-perm-fips-amd-f28
```

Comment:
This one is duplicated with `azure-ipi-des-mini-perm-arm`, we just one of them.

Reply:
removed azure-ipi-des-mini-perm-fips-amd-f28

Reply:
Correct me, I removed azure-ipi-des-mini-perm-arm